<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>611</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="distribution_graphpane_tab">
     <attribute name="title">
      <string>Распределение положений фотона</string>
     </attribute>
     <widget class="mplwidget" name="distribution_graph" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>591</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="density_histogram_tab">
     <attribute name="title">
      <string>Осевое сечение плотности распределения</string>
     </attribute>
     <widget class="mplwidget" name="density_graph" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>591</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="perform_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Провести исследование</string>
    </property>
   </widget>
   <widget class="QLabel" name="pix_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="main_window_resources.qrc">:/task_prefix/task.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>243</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="input_data_layout">
     <item>
      <layout class="QVBoxLayout" name="label_layout">
       <item>
        <widget class="QLabel" name="photons_quantity_label">
         <property name="text">
          <string>Количество фотонов (N)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="beam_radius_label">
         <property name="text">
          <string>Радиус пучка (a)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="spinBox_layout">
       <item>
        <widget class="QSpinBox" name="photons_quantity_spinBox">
         <property name="maximum">
          <number>100000000</number>
         </property>
         <property name="value">
          <number>1000000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="beam_radius_doubleSpinBox">
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mplwidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="main_window_resources.qrc"/>
 </resources>
 <connections/>
</ui>
